ir/Cargo.toml

```diff
@@ -1,7 +1,15 @@
 [package]
 name = "air-ir"
 version = "0.1.0"
+description="Intermediate represenation for the AirScript language"
```

Comment:
nit: There's a typo here. It should say "representation" instead of "represenation"